File: app/src/main/java/eu/kanade/tachiyomi/ui/reader/viewer/pager/horizontal/LeftToRightReader.java
package eu.kanade.tachiyomi.ui.reader.viewer.pager.horizontal;

public class LeftToRightReader extends HorizontalReader {

    @Override
    public void onFirstPageOut() {
        getReaderActivity().requestPreviousChapter();
    }

    @Override
    public void onLastPageOut() {
        getReaderActivity().requestNextChapter();
    }

}

File: app/src/main/java/eu/kanade/tachiyomi/ui/reader/viewer/pager/horizontal/LeftToRightReader.kt
package eu.kanade.tachiyomi.ui.reader.viewer.pager.horizontal

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import eu.kanade.tachiyomi.ui.reader.viewer.pager.PagerReader

/**
 * Left to Right reader.
 */
class LeftToRightReader : PagerReader() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedState: Bundle?): View? {
        return HorizontalPager(activity).apply { initializePager(this) }
    }

}
